src/SUMMARY.md

```diff
@@ -23,6 +23,9 @@
 - [Data analyses](data_analyses/README.md)
   - [differential expression analysis](data_analyses/differential_expression.md)
 
+- [Data downloads](data_downloads/README.md)
+  - [ncbi_datasets](data_downloads/ncbi-datasets-cli.md)
```

Comment:
Sorry to be nit-picking, but I'd actually go for underscores, here. I try to stick to [`snake_case` for naming things](https://r4ds.hadley.nz/workflow-basics.html#sec-whats-in-a-name), for readability and consistency. So:
```suggestion
  - [ncbi_datasets](data_downloads/ncbi_datasets_cli.md)
```

Reply:
You're definitely right, I too almost always use underscores but sometimes something just goes with the flow of your fingers...